applications/nrf5340_audio/doc/user_interface.rst

```diff
@@ -35,47 +37,50 @@ Buttons
-|               |                                                                                                           | * :ref:`nrf53_audio_unicast_client_app`     |
-|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
-|               | Pressed during playback: Mutes the playback volume.                                                       | * :ref:`nrf53_audio_unicast_server_app`     |
-|               |                                                                                                           | * :ref:`nrf53_audio_unicast_client_app`     |
-|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
-|               | Pressed on a BIS headset during playback: Change the gateway, if more than one is available.              | :ref:`nrf53_audio_broadcast_sink_app`       |
-+---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
-| **RESET**     | Resets the device to the originally programmed settings.                                                  | All                                         |
-|               | This reverts any changes made during testing, for example the channel switches with **VOL** buttons.      |                                             |
-+---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
++---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+-------------------------------+
+| Button        | Function                                                                                                  | Applications                                | Development Kit               |
+|               |                                                                                                           |                                             +------------------+------------+
+|               |                                                                                                           |                                             | nRF5340 Audio DK | nRF5340 DK |
++===============+===========================================================================================================+=============================================+==================+============+
+| **VOL-**      | Long-pressed during startup: Changes the headset to the left channel one.                                 | * :ref:`nrf53_audio_broadcast_sink_app`     |         Y        |      Y     |
```

Comment:
How do you want this to look @greg-fer: Now the buttons are in the leftmost column. We could delete that column all together and put the button names in the two rightmost columns.

Reply:
If we have time, I'd suggest using tabs for different kits like here https://github.com/nrfconnect/sdk-nrf/blame/main/samples/bluetooth/peripheral_uart/README.rst#L112 . We get less crowded tables and clear reference for each kit.

Reply:
I like that!